www/tasks/javascript.js

```diff
@@ -27,11 +29,20 @@ gulp.task('javascript', ['jshint', 'jscs'], function() {
 			loadMaps: true
 		}))
 		// Add transformation tasks to the pipeline here.
-		.pipe(uglify())
-		.pipe(sourcemaps.write('./'))
+		.pipe(gulpif(release, uglify()))
```

Comment:
We already have gutil as a dependency. We could easily use gutil.noop(), [linky](https://github.com/gulpjs/gulp-util)